lib/pronto/version.rb

```diff
@@ -0,0 +1,3 @@
+module Pronto
```

Comment:
Also you might be actually overriding pronto's version constant here. Would be better to nest into our own namespace: `Pronto::RailsMigrations::VERSION`.